gradle.properties

```diff
@@ -4,6 +4,7 @@ org.gradle.caching=true
 
 org.gradle.unsafe.configuration-cache=true
 
+systemProp.org.gradle.unsafe.kotlin.assignment=true
```

Comment:
❔ Do we need to add some comment here about what this is about and when it will be removed?

Reply:
I can add some.

Reply:
Added a comment